### DISCUSSION TITLE
NavigationMenuTrigger click instead of pointerenter / pointerleave

### DISCUSSION THREAD

**Discussion by lewebsimple** [unovue/reka-ui#676] [Q&A]
Similar to this [issue](https://github.com/radix-ui/primitives/issues/1630) on radix-ui, is it possible to configure NavigationMenuTrigger to open on click instead of mouse hover ?

**Comment by zernonia** [comment]
Relevant issue https://github.com/radix-vue/radix-vue/issues/565
